types/eth1.proto

```diff
@@ -161,6 +166,7 @@ message Eth1TransactionIndexed {
     bytes tx_fee = 8;
     bytes gas_price = 9;
     bool is_contract_creation = 10;
+    // unused, should mark reserved!
```

Comment:
**Nitpick:** This only relates to `invokes_contract`, right? Personally I would change this to
```suggestion
    // invokes_contract is unused, should mark reserved!
```
because for example here https://github.com/gobitfly/eth2-beaconchain-explorer/blob/27e923389afeca26ed17417fe933f7999048042d/types/eth1.pb.go#L1231-L1233 it looks like everything below that comment might be unused.

Also applies to https://github.com/gobitfly/eth2-beaconchain-explorer/blob/27e923389afeca26ed17417fe933f7999048042d/types/eth1.proto#L200.

Reply:
That's correct, I'll change it :)

Should we actually mark the field [reserved](https://protobuf.dev/programming-guides/proto3/#fieldreserved) though? So we have cleaned this up and prevent another dev in the future from relying on the value being set and correct

Reply:
@remoterami , I honestly don't have a strong opinion on that. Feel free to either resolve this comment without any further changes or to implement the changes.